src/main/scala/org/broadinstitute/dsde/firecloud/model/ElasticSearch.scala

```diff
@@ -68,6 +71,8 @@ object ESType extends ESPropertyFields {
 
 }
 
+case class ESNestedType(properties:Map[String,ESInnerField], `type`:String="nested") extends ESPropertyFields
```

Comment:
In the current case of parent label->order mapping, there is virtually no chance of having duplicate keys. This might not be true when re-used. Probably not worth worrying about for this PR.